dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -664,7 +665,7 @@ public void ContentFallbackFalse_PageFallbackFalse_PageEnglish_ViewEnglishConten
                         .setPageMode(PageMode.LIVE)
                         .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ENG = "+html , html.contains("content2content1"));
+        assertTrue("ENG = "+html , html.contains("content2content1"));
```

Comment:
Codacy found an issue: [The String literal "content2content1" appears 4 times in this file; the first occurrence is on line 668](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6309437)